.travis.yml

```diff
@@ -1,21 +1,16 @@
 sudo: false
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.2.3
-  - 2.2.4
+  - 2.2.5
   - 2.3.0
+  - 2.3.1
   - ruby-head
 matrix:
   allow_failures:
     - rvm: ruby-head
-    - rvm: 1.9.3
-    - rvm: 2.0
-    - rvm: 2.1
-    - rvm: 2.2
+    - rvm: 2.2.5
+    - rvm: 2.3.0
+    - rvm: 2.3.1
```

Comment:
Having 1.9.3 in the allow-failures section can be useful if someone wants to know what parts will fail on 1.9.3